contracts/token/psp1155/src/traits.rs

```diff
@@ -46,6 +46,8 @@ declare_storage_trait!(PSP1155Storage, PSP1155Data);
 #[derive(Default, Debug, SpreadLayout)]
 #[cfg_attr(feature = "std", derive(StorageLayout))]
 pub struct PSP1155MetadataData {
+    pub name: Option<String>,
```

Comment:
ERC1155 doesn't contain name and symbol, because it is multi token=) https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC1155/IERC1155.sol

Reply:
yeah my fault, removed